.github/pull_request_template.md

```diff
@@ -8,3 +8,13 @@
 
 Fixes #1
 -->
+
+## Tests
+
+<!-- We do not accept new features without corresponding automated tests. -->
+
+I've checked my work by:
+
+- [ ] Adding new test or adjusting existing tests
+- [ ] Testing manually in the browser
+- [ ] Looking at the code only
```

Comment:
What does this mean?

Reply:
The idea here is that people can put a `X` in the box, to say how they've tested their work.
The goal is for us to get a at-a-glance idea of how well tested the change is.

### Example: fixing typo in markdown file

Say I'd fixed a simple typo in the README, then I'd say:

> - [X] Looking at the code only

### Example: create new page

But if I made a new 404 page, I'd need to have:

> - [X] Adding new test or adjusting existing tests

### Example: new feature without tests

If I made a new feature and said:

> - [X] Looking at the code only

Then we could say: "Go write some tests".

Reply:
Oh, I see. I guess you see this as a single choice (radio buttons), but those are checkbox so I always think about this as non-excluding options.

I'd leave it to just `Adding new test or adjusting existing tests` and `Testing manually` for simplicity? So testing manually covers manual checks in the browser or checking something in the code. Then if there is a new feature added and the first checkbox is not check, we can say "hey this needs tests".

Reply:
Let's simplify then. 😄